admin/howto/manage-users.md

```diff
@@ -0,0 +1,45 @@
+# Manage access to the hub
+
+The **Administrator Panel** can be used to maintain the list of users
+who are authorized to use your hub. You can access this panel by clicking
+the 'Admin' button in the top bar in your hub control panel.
```

Comment:
Here's an example where we could de-duplicate this sentence by just having a section called "The Admin Panel" with instructions on how to get there, then we could just link to that?